vm/devices/storage/disk_nvme/nvme_driver/src/driver.rs

```diff
@@ -364,9 +423,11 @@ impl<T: DeviceBacking> NvmeDriver<T> {
         self.admin = Some(admin.issuer().clone());
 
         // Pre-create the IO queue 1 for CPU 0. The other queues will be created
-        // lazily.
+        // lazily. Numbering for I/O queues starts with 1 (0 is Admin).
+        let qid1 = worker.io.len() as u16 + 1;
```

Comment:
Isn't this always 1? Can we just write 1?

Reply:
It is always 1. I thought the logic is easier to understand this way, can update.

Reply:
I'd prefer we leave any code the same that doesn't need to change, just to make the change easier to reason about.